source/uwp/UWPTestLibrary/TestResultViewModel.cs

```diff
@@ -488,7 +492,14 @@ public override string ToString()
             }
             else if (NewCard == true)
```

Comment:
```suggestion
            else if (NewCard)
```

not your bug, but... :)

Reply:
Probably is my bug, to be honest :)

Fixed.

---
In reply to: [366065171](https://github.com/microsoft/AdaptiveCards/pull/3724#discussion_r366065171) [](ancestors = 366065171)